tools/download_counters/README.md

```diff
@@ -0,0 +1,44 @@
+# Download Counters Script
+
+This script retrieves the download counts of a list of repositories from the GitHub API.
+
+## Prerequisites
+
+- Bash
+- `curl`
+- `jq` (for parsing JSON responses)
+
+## Setup
+
+1. Install `jq` if you don't have it already:
+
+   ```bash
+   sudo apt-get install jq
+   ```
+
+2. Obtain a GitHub personal access token. You can create one [here](https://github.com/settings/tokens).
```

Comment:
maybe here, a lack of information to configure the personal github token, i.e. options to select for token generation (after testing your script, only "repo / public_repo" option needed)